cypher/cypher-docs/src/docs/dev/execution-plans-operator-summary.asciidoc

```diff
@@ -651,6 +651,18 @@ Tests for the absence of a pattern predicate if an expression predicate evaluate
 |
 |
 
+| <<query-plan-directed-union-relationship-types-scan,Directed Union Relationship Types Scan>>
```

Comment:
`<<query-plan-directed-union-relationship-types-scan,DirectedUnionRelationshipTypesScan>>`

Reply:
Also this is placed on the wrong place, this page has the operators in sorted order.